src/plugins/custom_import_map/public/components/__snapshots__/vector_upload_options.test.tsx.snap

```diff
@@ -210,6 +210,11 @@ Object {
                   </span>
                 </button>
               </div>
+              <div
+                aria-label="medium-spacer"
+                class="euiSpacer euiSpacer--m"
+              />
+              <div />
```

Comment:
why an extra div here?

Reply:
that is auto generated snapshot, it's part of the snapshot test.